example/example.js

```diff
@@ -0,0 +1,23 @@
+const { AudioProcessor } = require('../index'); // Assuming your index.js is in the parent directory
```

Comment:
should just be
```js
const { AudioProcessor } = require('..');
```
because index.js should be in package.json the the src file

Reply:
Oh right. require('..') works, didn't realise that package.json is already including index.